### PR TITLE
[BE] Normalize endpoints language

### PR DESCRIPTION
**Pull request by alelnro1**
2023-10-30 - 14-43-20 - Swagger UI

### CODE REVIEW

**Comment by alelnro1** [comment]
@dody87 Fixed merge conflicts

**Review by BrunoMPedraza — APPROVED** [review]

**Comment by alelnro1** [comment]
Conflicts are solved

File: api/swagger.json [hunk 1]
```diff
@@ -73,7 +73,72 @@
         }
       }
     },
-    "/mesa": {
+    "/voting-tables/{id}/reportarFaltaFiscal": {
```
Comment (brunocalmels) [inline api/swagger.json:76]: Shouldn't all the endpoints be in English (following the nature of this PR)?

**Review by brunocalmels — COMMENTED** [review]
Also... how breaking is this for anyone else working on these enpoints?

**Comment by brunocalmels** [comment]
There are still merge conflicts @alelnro1

**Comment by alelnro1** [comment]
@brunocalmels Conflicts are solved. Regarding the breaking changes, here is a screenshot of all the working API endpoints
2023-10-30 - 18-34-16 - Swagger UI

File: api/swagger.json (hunk 1 above)
Comment (alelnro1) [reply api/swagger.json]: @brunocalmels Solved

**Review by brunocalmels — APPROVED** [review]

**Comment by brunocalmels** [comment]
LGMT now

**Pull request merged**